tests/general/InvalidLinkBearTest.py

```diff
@@ -203,3 +203,27 @@ def test_pip_vcs_url(self):
 
         for line in brokenlink_at_hash.splitlines():
             self.assertResult(invalid_file=[line])
+
+    def test_links_to_ignore(self):
+        valid_file = """http://example.com/404
+                        http://localhost:8080
+                        http://httpbin.org/status/500
+                        http://www.notexample.com/404""".splitlines()
+
+        invalid_file = """http://example.com/404
+                          http://localhost:8080
+                          http://httpbin.org/status/500
+                          http://httpbin.org/status/503
+                          http://www.notexample.com/404""".splitlines()
+
+        link_ignore_list = [
+                           'localhost:8080',
+                           'http://httpbin.org/status/500',
+                           'http://www.notexample.com/404',
+                           'http://example.com/404',
+                           'http://example.co.in/404'
+                          ]
+
+        self.assertResult(valid_file=valid_file,
+                          invalid_file=invalid_file,
+                          settings={'link_ignore_list': link_ignore_list})
```

Comment:
this test doesn't test properly. Read the other tests please and understand how this is done.